### DISCUSSION TITLE
Ways to run tests with the Launcher API

### DISCUSSION THREAD

**Discussion by ghost** [junit-team/junit-framework#3582] [Q&A]
Is there any built-in ways to run tests with the Launcher API on the JUnit Platform?
I mean without manually starting the main() method as shown in the User Guide examples.

And is there any support from Intelij IDEA when running such tests?
Just like what happens when running tests through the IDE functionality.
![image](https://github.com/junit-team/junit5/assets/120301053/688b3b73-6c76-4784-a6a2-77f1318db954)

**Comment by marcphilipp** [comment]
Please elaborate on what you'd like to achieve by running tests via the Launcher API. IntelliJ IDEA is already doing that behind the scenes.
